pontoon/base/models.py

```diff
@@ -1854,6 +1899,11 @@ def for_project_locale(self, project, locale, paths=None, statuses=None,
         pre_filters = []
         post_filters = []
 
+        # Temporary optimization: use simpler status & extra filter DB queries for non-PO projects,
+        # which don't store translations for different plural forms in separate Translation model
+        # instances.
+        no_plurals = True if len(project.resources.filter(format='po')) == 0 else False
```

Comment:
`no_plurals = (len(project.resources.filter(format='po')) == 0)`

Reply:
You can even do without the parentheses. And you can also do this: 
```python
no_plurals = not len(project.resources.filter(format='po'))
```

Or you can go further and do this (but I'm not sure that's more readable):
```python
no_plurals = not project.resources.filter(format='po')
```

Reply:
Or even .exists(), https://docs.djangoproject.com/en/1.8/ref/models/querysets/#django.db.models.query.QuerySet.exists.